examples/tutorials/advanced/contour_map.py

```diff
@@ -65,7 +65,7 @@
 # ------------
 #
 # The :meth:`pygmt.Figure.grdcontour` method accepts additional parameters,
-# including setting the projection and frame.
+# including setting projection and frame.
```

Comment:
I'm not sure why "the" should be removed here.

Reply:
For me, this sounds better 🙂. But if the others agree that the current version is better/correct, I am fine with changing this back.

Reply:
Since "projection" and "frame" are both singular, I would think it would make more sense to add "the".

Reply:
Agree Witz @willschlitzer to leave the "the" as is.

Reply:
Thanks for your answers! Changed it back in commit  https://github.com/GenericMappingTools/pygmt/pull/2190/commits/9c76c2d062abd345790c617d19147e8d63e74d31.